Yosemite/Yosemite/Actions/ProductAction.swift

```diff
@@ -107,10 +107,10 @@ public enum ProductAction: Action {
     ///
     case replaceProductLocally(product: Product, onCompletion: () -> Void)
 
-    /// Checks if the store is eligible for products onboarding.
+    /// Checks if the store already has any products.
     /// Returns `true` if the store has no products.
```

Comment:
Same here. I think we have to update this comment.

Reply:
Also updated in 339bae8c8a10318bd06041ed06baed8dc30129ae.